app/frontend/stylesheets/all/header.scss

```diff
@@ -23,6 +23,12 @@ body.production {
   }
 }
 
+body.development {
+  .main-nav {
+    @extend .bg-secondary;
+  }
+}
+
```

Comment:
Are we trying to change the main-nav colour for each environment?
Why are we doing this only for the development environment here?
If we do this, doesn't it mean that we should do it for each environment?

Reminders:

In the existing code, we have the main-nav as red (for anything), we override it only for production by extending bg-primary and turn it to blue. See `app/frontend/stylesheets/all/header.scss`

In the existing code, logo_container/app_name div, the text below logo is for recognising the environment. See `app/views/shared/_header.html.erb`

Reply:
This is way better. The colours chosen are easy to remember. Thanks.